ConwaysGameOfLife.Tests/GameControllerShould.cs

```diff
@@ -23,8 +23,10 @@ public GameControllerShould()
         public void EndRunImmediately_GivenQInput()
         {
             var expectedOutput = "Game of Life has ended.";
+            
             var input = "q";
-            _input.GetReadLine(input);
+            _input.GetReadLine("3,3");
+            _input.GetReadKey(input);

```

Comment:
This change makes it so the test is checking if you can quit after a `Universe` has been created and the game is now prompting for a `Location` i.e. the test is no longer reflecting what the name implies. It's more or less doing the same as the `EndGame_GivenQInputDuringLocationInput` test in terms of behaviour